src/org/omegat/Bundle.properties

```diff
@@ -2906,12 +2906,15 @@ PREFS_PLUGINS_COL_VERSION=Version
 PREFS_PLUGINS_COL_AUTHOR=Author
 PREFS_PLUGINS_COL_DESCRIPTION=Description
 PREFS_PLUGINS_INSTALL_FROM_DISK=&Install plugin from disk
-PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION=Install plugin file
-PREFS_PLUGINS_CONFIRM_UPGRADE=Upgrade/override plugin {0}\n\
-   current: {1}\n\
-   new version: {2}
-PREFS_PLUGINS_CONFIRM_INSTALL=Install plugin {0} {1}
+PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION=Install Plugin
```

Comment:
I would also like to add a key for an upgrade dialog message:
`PREFS_PLUGINS_TITLE_CONFIRM_UPGRADE=Upgrade Plugin`

I not confident about how to modify `PluginInstaller.java` to implement that change.

I think I would have to add an extra line of code to assign the title above the `message=...` statement in line 103 and line 106 in the current file?

At the current line 103:

```java
title = PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION
message = StringUtil.format(OStrings.getString("PREFS_PLUGINS_CONFIRM_OVERWRITE"), pluginName,
                        currentInfo.getVersion(), version);
```

At the current line 106:

```java
title = PREFS_PLUGINS_TITLE_CONFIRM_UPGRADE
message = StringUtil.format(OStrings.getString("PREFS_PLUGINS_CONFIRM_UPGRADE"), pluginName,
                        currentInfo.getVersion(), version);
```

And then change line 117 to:
```java
OStrings.getString(title), JOptionPane.OK_CANCEL_OPTION,
```

Am I on the right track?

Reply:
It is better that make line 106  `title = OStrings.getString("PREFS_PLUGINS_TITLE_CONFIRM_UPGRADE")` then line 117 to `title, JOptionPane.OK_CANCEL_OPTION,` because OmegaT project test set checks `OStrings.getString("PREFS_PLUGINS_TITLE_CONFIRM_UPGRADE")` syntax and distill key and check Bundles for missing keys.

Reply:
OK, thanks!

By the same logic, should line 103 then be `title = OStrings.getString("PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION")?`